warehouse/query-core/src/main/java/datawave/query/tld/TLD.java

```diff
@@ -23,112 +19,202 @@
  * </ul>
  */
 public class TLD {
+    private final static char NULL_BYTE = '\u0000';
+    private final static char DOT = '.';
```

Comment:
Sonar lint is suggesting the following
```suggestion
    private static final char NULL_BYTE = '\u0000';
    private static final char DOT = '.';
```

Reply:
done